client/my-sites/site-settings/settings-newsletter/main.tsx

```diff
@@ -279,9 +282,10 @@ const NewsletterSettingsForm = wrapSettingsForm( getFormSettings )( ( {
 			/>
 			<Card className="site-settings__card">
 				<FeaturedImageEmailSetting
-					disabled={ disabled }
+					disabled={ disabled || isPrivate }
```

Comment:
Same comment as the Jetpack version. Would we want to default the check to OFF if it's disabled? Could be slightly confusing otherwise. When its disabled in the off position it would lead me to believe its stuck ON despite the message we have.

Reply:
Happy to do that. I just pushed 0226088e8d6ee585f964cf5dc75ae30845efaaef